src/emmeans.jl

```diff
@@ -73,13 +82,28 @@ function emmeans(model::RegressionModel; eff_col=nothing, err_col=:err,
     if !isnothing(dof)
         result[!, :dof] .= _dof(dof, model)
     end
+    if !isnothing(ci_level)
+        # don't include this in the above if because
+        # we don't want to potentially add a dof column if there is no CI
+        if isnothing(dof)
+            # fall back to z value
+            result[!, :dof] .= Inf
+        end
+        # divide by two for twosided
+        # 1+level to pull from upper tail
+        scale = quantile.(TDist.(result[!, :dof]), (1 + ci_level) / 2)
```

Comment:
Should this be 1- instead of 1+?
```suggestion
        # 1-level to pull from upper tail
        scale = quantile.(TDist.(result[!, :dof]), (1 - ci_level) / 2)
```

Reply:
hmmm, yeah, I think so but we to add an abs (because the lower tail is negative)

Reply:
```suggestion
        scale = abs.(quantile.(TDist.(result[!, :dof]), (1 - ci_level) / 2))
```